src/scribe_data/utils.py

```diff
@@ -152,46 +198,22 @@ def get_language_words_to_remove(language: str) -> list[str]:
 
     Returns
     -------
-        The words that should not be included as autosuggestions for the given language as values of a dictionary.
+        list[str]
+            The words that should not be included as autosuggestions for the given
+            language
     """
-    language = language.lower()
-    words_to_remove: dict[str, list[str]] = {
-        "english": [
-            "of",
-            "the",
-            "The",
-            "and",
-        ],
-        "french": [
-            "of",
-            "the",
-            "The",
-            "and",
-        ],
-        "german": ["of", "the", "The", "and", "NeinJa", "et", "redirect"],
-        "italian": ["of", "the", "The", "and", "text", "from"],
-        "portuguese": ["of", "the", "The", "and", "jbutadptflora"],
-        "russian": [
-            "of",
-            "the",
-            "The",
-            "and",
-        ],  # and all non-Cyrillic characters
-        "spanish": ["of", "the", "The", "and"],
-        "swedish": ["of", "the", "The", "and", "Checklist", "Catalogue"],
-    }
-
-    if language not in words_to_remove:
-        raise ValueError(
-            f"{language.capitalize()} is currently not a supported language."
-        )
-
-    return words_to_remove[language]
+    return _find(
+        "language",
+        language,
+        "remove-words",
+        f"{language.capitalize()} is currently not a supported language.",
+    )
 
 
 def get_language_words_to_ignore(language: str) -> list[str]:
     """
```

Comment:
@andrewtavis : This function has the same description as `get_language_words_to_remove` on line 189. This _could_ be an error. One this is rectified the entry on line 9 of `src/scribe_data/resources/language_meta_data.json` can be updated.